www/docs/externals.md

```diff
@@ -58,9 +66,73 @@ Encapsulating external reactive dependencies as eager collections, that complex
 ## Non-Skip services
 
 Of course, unless your application is built from the ground up using the Skip framework, it is likely that your application depends on some non-reactive external system: REST APIs, databases, external HTTP endpoints, and the like.
-These systems operate on a pull-based request/response paradigm, so some work is required to adapt them to Skip's eager push-based paradigm.
+
+### PostgreSQL
+
+One common use case for Skip is to reactively update and push results in response to updates in a source-of-truth relational database.
```

Comment:
Is "source-of-truth" relevant here? The db might just as well not be the source of truth from our perspective, as long as the SoT is not us.